webapp/components/Comment.vue

```diff
@@ -143,26 +142,14 @@ export default {
     },
     async deleteCommentCallback() {
       try {
-        await this.$apollo.mutate({
+        const {
+          data: { DeleteComment },
+        } = await this.$apollo.mutate({
           mutation: CommentMutations(this.$i18n).DeleteComment,
           variables: { id: this.comment.id },
-          update: async store => {
-            const data = await store.readQuery({
-              query: PostQuery(this.$i18n),
-              variables: { id: this.post.id },
-            })
-
-            const index = data.Post[0].comments.findIndex(
-              deletedComment => deletedComment.id === this.comment.id,
-            )
-            if (index !== -1) {
-              data.Post[0].comments.splice(index, 1)
-            }
-            await store.writeQuery({ query: PostQuery(this.$i18n), data })
-          },
```

Comment:
removing this doesn't reintroduced the bug that if a user deletes a comment, then edits another comment, the deleted comment get added back to the comment list?

Reply:
![Screenshot - 2019-09-03T145956 292](https://user-images.githubusercontent.com/2110676/64175280-42d34a80-ce5b-11e9-99b3-bf0697aeaa47.png)
could not reproduce